src/main/java/frc/robot/util/DriverUI.java

```diff
@@ -16,6 +16,9 @@ public class DriverUI implements Loggable {
     @Log(name="Select Auto:", rowIndex = 0, columnIndex = 4, height = 1, width = 2)
     public static SendableChooser<AutoPose> autoChooser = new SendableChooser<>();
     
+    @Log(name="Field", rowIndex = 0, columnIndex = 0, height = 6, width = 6)
```

Comment:
I would make this field the same field as the one in the earlier version of the file. This ensures that there isn't any scaling issues.
```java
@Log.Field2d(name = "Field", rowIndex = 1, columnIndex = 0, height = 4, width = 10)
    public static Field2d field = new Field2d();
```